client/_locales/en/messages.json

```diff
@@ -1,66 +1,66 @@
 {
   "l10nTabName": {
-    "message":"Localization"
-    ,"description":"name of the localization tab"
-  }
-  ,"l10nHeader": {
-    "message":"It does localization too! (this whole tab is, actually)"
-    ,"description":"Header text for the localization section"
-  }
-  ,"l10nIntro": {
-    "message":"'L10n' refers to 'Localization' - 'L' an 'n' are obvious, and 10 comes from the number of letters between those two.  It is the process/whatever of displaying something in the language of choice.  It uses 'I18n', 'Internationalization', which refers to the tools / framework supporting L10n.  I.e., something is internationalized if it has I18n support, and can be localized.  Something is localized for you if it is in your language / dialect."
-    ,"description":"introduce the basic idea."
-  }
-  ,"l10nProd": {
-    "message":"You <strong>are</strong> planning to allow localization, right?  You have <em>no idea</em> who will be using your extension!  You have no idea who will be translating it!  At least support the basics, it's not hard, and having the framework in place will let you transition much more easily later on."
-    ,"description":"drive the point home.  It's good for you."
-  }
-  ,"l10nFirstParagraph": {
-    "message":"When the options page loads, elements decorated with <strong>data-l10n</strong> will automatically be localized!"
-    ,"description":"inform that <el data-l10n='' /> elements will be localized on load"
-  }
-  ,"l10nSecondParagraph": {
-    "message":"If you need more complex localization, you can also define <strong>data-l10n-args</strong>.  This should contain <span class='code'>$containerType$</span> filled with <span class='code'>$dataType$</span>, which will be passed into Chrome's i18n API as <span class='code'>$functionArgs$</span>.  In fact, this paragraph does just that, and wraps the args in mono-space font.  Easy!"
-    ,"description":"introduce the data-l10n-args attribute.  End on a lame note."
-    ,"placeholders": {
+    "message": "Localization",
```

Comment:
When a locale is specified in manifest.json, this file is required. All I did was to reformat it from the generated code